src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo__link">
+        <a href="/">
+          <img
+            src="images/logo.png"
+            alt="Company logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list__item">
```

Comment:
Fix classes everywhere
```suggestion
          <li class="nav__item">
```